packages/ui-focusable/src/Focusable/index.tsx

```diff
@@ -266,13 +258,16 @@ class Focusable extends Component<FocusableProps> {
       const rendered = render({
         focused,
         focusable,
-        focusVisible: this.isFocusVisible(focusable, focused),
+        focusVisible: this.isFocusVisible(focused, focusable),
         attachRef: this.attachRef
       })
-
+      type ElementWithRef = { ref?: () => unknown }
```

Comment:
I would probably expand the type in the `props.ts`, but not sure that's the better approach.

```
type FocusableRenderFunction = (
  opts: FocusableRenderOptions
) => (
  | ReactElement
  | DOMElement<any, any>
  | CElement<any, any>
  | FunctionComponentElement<any>
) & { ref?: () => unknown }
```

Actually, only `ReactElement` doesn't have this prop, and all the others (`CElement` and `FunctionComponentElement` and `DOMElement`) are extending it, so is it needed?

Reply:
sadly that's still not enough becuase `ref` needs to be a function, and sometimes its not one. I've added the ref to props.